packages/broker/test/integration/plugins/storage/Storage.test.ts

```diff
@@ -48,20 +47,18 @@ function buildEncryptedMsg({
     timestamp,
     sequenceNumber,
     publisherId = publisherZero,
-    msgChainId = '1',
-    content = 'ab3516983712fa4eb216a898ddd'
+    msgChainId = '1'
 }: {
     streamId: string
     streamPartition: number
     timestamp: number
     sequenceNumber: number
     publisherId?: EthereumAddress
     msgChainId?: string
-    content?: string
```

Comment:
Could also remove the `content = {}` from line 27 so that the two functions are kept in sync.